components/automate-ui/src/app/entities/policies/policy.selectors.ts

```diff
@@ -16,15 +16,9 @@ export const iamMajorVersion = createSelector(
   (state) => state.iamMajorVersion
 );
 
-export const iamMinorVersion = createSelector(
+export const isIAMv2 = createSelector(
```

Comment:
Most of the UI code uses not-equals-v1 rather than equals-v2 to allow for future expansion. The latter would make things stop working when v3 comes along, which will probably not be what we want.
Thus, would suggest either make this `isIAMv1` or rename to `atLeastV2`, depending on what polarity you want.
Then L21 would compare to 'v1' in either case.

Reply:
Thank you for the reminder to search for `!== 'v1'`! Will do.

I'd like to leave the equality for now. We don't know what v3 looks like, yet, so I'm not confident we won't need to modify this variable anyway.